### ISSUE TITLE
Contact for accessibility questions

### ISSUE THREAD

**Issue by AJSMITH1512** [nhsuk/nhsuk-service-manual#960]
### What

Can we consider what we have on the accessibility pages for contact us.

### Why 

- Should this be the Service Manual Team or accessibility team slack channel?

Accessibility group own pages and should be supporting the wider population we do not have accessibility specialist embedded in the team. 

### To do

- [x]  Arrange discussion
- [x] Recommend way forward

**Comment by AJSMITH1512** [comment]
This is now being picked up as part of the community work - how we want the community to contact us about different things.

Does accessibility need to be a different contact? Is the Service Standard also a different contact?

Accessibility Working Group

NHSx - Assessment Team

**Comment by Tosin-Balogun** [comment]
Thanks for this @AJSMITH1512. Do we have the contact details for the accessibility working group?

**Comment by sarawilcox** [comment]
Linked to 2 epics:
- [EPIC: Improve Service Manual website contribution user experience}(https://github.com/nhsuk/nhsuk-service-manual/issues/993)
- [EPIC: Accessibility guidance (update + assess existing guidance)](https://github.com/nhsuk/nhsuk-service-manual/issues/669)

**Comment by sarawilcox** [comment]
We talked about routing users to GitHub issues for the pages in the accessibility guidance so that they can share their findings/comments in one place. We'll need to make sure that the AWG looks at the GitHub issues. Similarly with service standard pages and the NHSX Assessment Team.

**Comment by sarawilcox** [comment]
We have GitHub issues for all the accessibility pages now and Michael Cheung will pick up any issues. As the AWG isn't meeting at the mo, I think that's the best we can do. People always have the option of the service manual email.

![Screenshot 2021-06-25 at 09 34 50](https://user-images.githubusercontent.com/45792959/123396002-b8f8f200-d598-11eb-92c1-5739b1989ce5.png)

I can also pick this up in the work I'm going to do on the call out actions at the bottom of different sections - explaining how we use Slack.

**Comment by sarawilcox** [comment]
Perhaps hold this open till we can link it to the new issue?
